doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -983,7 +983,9 @@ zcbor
 Trusted Firmware-M
 ==================
 
-|no_changes_yet_note|
+* Added:
+
+  * Internal Trusted Storage (ITS) with enrcyption, see :ref:`tfm_encrypted_its`
```

Comment:
You can consider saying 
* Section :ref:... describing Internal Trusted Storage (ITS) with encryption.

Reply:
Changed it